docs/bosh/main.tf

```diff
@@ -74,12 +81,14 @@ resource "google_compute_firewall" "bosh-bastion" {
   }
 
   target_tags = ["bosh-bastion"]
+  project     = "${var.xpn_host_projectid}"
```

Comment:
No, just firewall rule. The NATS VM does get created in the XPN project so we can setup routes to it. The rest are in the client project.